readme.html

```diff
@@ -13,4 +13,19 @@
 and limitations under the License.
 -->
 
-<p>Elasticsearch installations can be configured to allow REST API access without any type of authentication. The app therefore marks <b>username</b> and <b>password</b> as optional parameters. If specified, the app will end up using these for generating the basic authentication header used in the various Elasticsearch REST endpoints.<br>Connection can be configured over HTTP or HTTPS, so if <b>test connectivity</b> fails please check the protocol.</p>
+<p>Elasticsearch installations can be configured to allow REST API access without any type of authentication.
+    The app therefore marks <b>username</b> and <b>password</b> as optional parameters. If specified, the app
+    will end up using these for generating the basic authentication header used in the various Elasticsearch
+    REST endpoints.<br>Connection can be configured over HTTP or HTTPS, so if <b>test connectivity</b>
+    fails please check the protocol.
+</p>
+<p><b>Playbook Backward Compatibility</b><br>
+    Action parameter has been removed to the action given below. Hence, It is requested to the end-user please update their existing playbooks parameters to ensure the correct functioning
+    of the playbooks created on the earlier versions of the app.
+    <ul>
+        <li><b>For version 2.1.0 :</b></li>
```

Comment:
Is the "version 2.1.0" referencing the last version of the Elasticsearch connector that was released? According the previous release notes, the last released version was 2.0.5 not 2.1.0. I would suggest something like this, assuming the new release will be v3.0.0 (as defined in the JSON file):

```suggestion
        <li><b>Changes from version 2.0.5 to version 3.0.0:</b></li>
```

Reply:
Yes it should be v3.0.0, updated the changes